railties/test/application/configuration_test.rb

```diff
@@ -2905,6 +2905,12 @@ class MyLogger < ::Logger
       assert_not_includes(output, "rails_direct_uploads")
     end
 
+    test "ActiveStorage.video_preview_arguments can be configured" do
+      app "development"
+
+      assert_equal ActiveStorage.video_preview_arguments, "-y -vframes 1 -f image2"
```

Comment:
This isn't really testing that it can be configured, just that it can be read. Can we change the setting in an  initializer and check that it's changed from the default, thanks.

Reply:
No worries :) I added a test to ensure a changed config works as well as a test for the default.